configs/cuml_config.json

```diff
@@ -0,0 +1,486 @@
+{
```

Comment:
I would put it in folders:
- sklearn
- xgboost
- cuml
...

Reply:
Not in scope of this PR